src/server/usage/classes/collector_set.js

```diff
@@ -94,20 +94,6 @@ export class CollectorSet {
     return Promise.all(fetchPromises);
   }
 
-  bulkFormat(data) {
```

Comment:
It didn't make a lot of sense to have this in OSS; I moved it to `BulkUploader.toBulkUploadFormat `